arborist/server.go

```diff
@@ -1260,12 +1260,14 @@ func (server *Server) handleGroupCreate(w http.ResponseWriter, r *http.Request,
 		_ = response.write(w, r)
 		return
 	}
-	var errResponse *ErrorResponse
-	if r.Method == "PUT" {
-		errResponse = transactify(server.db, group.overwriteInDb)
-	} else {
-		errResponse = transactify(server.db, group.createInDb)
-	}
+	authzProvider := getAuthZProvider(r)
+	errResponse := transactify(server.db, func(tx *sqlx.Tx) *ErrorResponse{
```

Comment:
looooooool
I started this https://play.golang.org/p/kuLvJBd8lNL , don't know how to get past the type error but maybe you do? If you like the general idea, that is

Reply:
go T^T